cmd/tiller/release_server_test.go

```diff
@@ -667,7 +665,7 @@ func TestUninstallPurgeDeleteRelease(t *testing.T) {
 	}
 
 	_, err2 := rs.UninstallRelease(c, req2)
-	if err2 != nil {
+	if err2 != nil && err2.Error() != "'angry-panda' has no deployed releases" {
```

Comment:
Can we avoid error assertions by string? Can we create a new error type instead?